app/src/i18n/locales/en/data.json

```diff
@@ -579,11 +578,14 @@
   "fugitive-emissions-oil-gas-type-oil": "Oil",
   "gallons-per-kilometer": "Gallons per kilometer (gal/km)",
   "gallons-per-miles": "Gallons per miles (gal/mi)",
-  "incineration-waste-inboundary-boiler-type": "Boiler type",
+  "boiler-type": "Boiler type",
```

Comment:
We need to change these keys back since the ID change for the inputs was reverted, right?

Reply:
For all 3 affected translation files